ruby/2.6.5/Dockerfile

```diff
@@ -0,0 +1,103 @@
+FROM socrata/base
+MAINTAINER Socrata <sysadmin@socrata.com>
+
+# skip installing gem documentation
+RUN mkdir -p /usr/local/etc \
+	&& { \
+		echo 'install: --no-document'; \
+		echo 'update: --no-document'; \
+	} >> /usr/local/etc/gemrc
+
+ENV RUBY_MAJOR 2.6
+ENV RUBY_VERSION 2.6.5
+ENV RUBY_DOWNLOAD_SHA256 d5d6da717fd48524596f9b78ac5a2eeb9691753da5c06923a6c31190abe01a62
+ENV RUBYGEMS_VERSION 3.0.4
+ENV BUNDLER_VERSION 2.0.2
```

Comment:
we may not need to do this for 2.6+ as I thought bundler was included in the distribution by default now.

Reply:
it looks like it's included, but it's installing `1.17.2` on my machine.

Reply:
ok then lets stick with that.